runtime/doc/diagnostic.txt

```diff
@@ -937,6 +937,13 @@ show({namespace}, {bufnr}, {diagnostics}, {opts})
       • {opts}         (`vim.diagnostic.Opts?`) Display options. See
                        |vim.diagnostic.Opts|.
 
+status()                                             *vim.diagnostic.status()*
+    Returns formatted string with diagnostics for the current buffer. Empty
+    diagnostics are not included.
```

Comment:
Something more in the line of "The severities with 0 diagnostics are left out."? Empty sounds weird here, they're not 'empty', but rather absent. Would also be nice to mention the use of `vim.diagnostic.signs.text` and show an example (`E:1 I:2`).